chore: update op network asset

artsiomYavorski opened the pull request:
## Issue

Link to Trello card

## Description

What this does and why it's needed, goals of the pull request's commits

## Verifying

How to confirm changes, and explain how this was tested

_Steps to verify_

_Expected results_

## Implementation details

Highlights of how the implementation fixes the issue or anything relevant about a new feature

## Screenshots

### Before

### After

## Related PRs

List related PRs against other branches:

| Short identifier    | Source                  |
| ------------------- | ----------------------- |
| back-end-related-pr | [link](paste-link-here) |
| some-other_pr       | [link](paste-link-here) |

## Todos (if a PR is in work in progress state)

- [ ] Tests
- [ ] Documentation
- [ ] Other things can be put here

dimlbc commented:
Doesn't work in lts and yield apps.
<img width="535" alt="Screenshot 2025-02-23 at 16 55 00" src="https://github.com/user-attachments/assets/22159d39-82f7-4f43-ba7b-7b237b23fe19" />
<img width="367" alt="Screenshot 2025-02-23 at 16 55 31" src="https://github.com/user-attachments/assets/92afbb2e-a132-4fe7-ab75-1281b761c6af" />

Also if it is possible let's update in rainbowkit modal (not critical)
<img width="437" alt="Screenshot 2025-02-23 at 16 56 28" src="https://github.com/user-attachments/assets/617be744-4131-49e1-948d-232568d5e23b" />

artsiomYavorski commented:
> Doesn't work in lts and yield apps. <img alt="Screenshot 2025-02-23 at 16 55 00" width="535" src="https://private-user-images.githubusercontent.com/49659185/416001508-22159d39-82f7-4f43-ba7b-7b237b23fe19.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDAzNDU1ODUsIm5iZiI6MTc0MDM0NTI4NSwicGF0aCI6Ii80OTY1OTE4NS80MTYwMDE1MDgtMjIxNTlkMzktODJmNy00ZjQzLWJhN2ItN2IyMzdiMjNmZTE5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjIzVDIxMTQ0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU0NTFlMWE2NDEwODJkYzhkNjhlYzMyN2M4NThiMGE1MTNkNzQ2MjRmN2NhYzYxNzVkYTFjYmMyNjlmYjM2MzAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.cKIZqhP2nuZRhyelYCwhDkQknXH0XqAQfhFA3m0u53o"> <img alt="Screenshot 2025-02-23 at 16 55 31" width="367" src="https://private-user-images.githubusercontent.com/49659185/416001512-92afbb2e-a132-4fe7-ab75-1281b761c6af.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDAzNDU1ODUsIm5iZiI6MTc0MDM0NTI4NSwicGF0aCI6Ii80OTY1OTE4NS80MTYwMDE1MTItOTJhZmJiMmUtYTEzMi00ZmU3LWFiNzUtMTI4MWI3NjFjNmFmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjIzVDIxMTQ0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTcxOTFmMmVmYWFlOGU4YTEzMDRmZmY2N2M4NjE1MTkzNTYwNzIzZTMyMjc5ZWQyMDk3OWQwYmQ2ZGM0OTZkZDcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.-WT86j75EhNCfuNnQtGY-ZPfLeQIla9tb1WCqaisT80">
> 
> Also if it is possible let's update in rainbowkit modal (not critical) <img alt="Screenshot 2025-02-23 at 16 56 28" width="437" src="https://private-user-images.githubusercontent.com/49659185/416001533-617be744-4131-49e1-948d-232568d5e23b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDAzNDU1ODUsIm5iZiI6MTc0MDM0NTI4NSwicGF0aCI6Ii80OTY1OTE4NS80MTYwMDE1MzMtNjE3YmU3NDQtNDEzMS00OWUxLTk0OGQtMjMyNTY4ZDVlMjNiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjIzVDIxMTQ0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE4NzQwNjNhZWM5YmFjYTk3MmE4NzE1NmQ5MzAyODFjOGE2NTVlYmNiMTQ0YTM4ZGU1ZTE1ZjY2NWQ5NmE0MzkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.7c98jfRFOWVCxVcvRFwII6PxCqgeDuflyywdFc5Or0A">

Missed updating of view box. fixed.
Added rainbow kit customisation.
